### ISSUE TITLE
Simple Camera That Looks At Nearest Player

### ISSUE THREAD

**Issue by Konclan** [TeamSpen210/BEE2-items#912]
The monitor cameras are nice; however, I would like to just put some as aesthetics and want them to look at the nearest player rather than a single marked location. This could be an option on the normal camera or a different item.

**Comment by ClarkxlYT** [comment]
this is not a issue lol

**Comment by ZombiePwner1** [comment]
@ClarkxlYT This is a issue just because it says issue doesn't mean it's a bug report it can be enhancements new item reports etc.

**Comment by ClarkxlYT** [comment]
@ZombiePwner1 it is not a issue it is a pull request

**Comment by Konclan** [comment]
Why did you change the title? The label makes it redundant.

**Comment by ZombiePwner1** [comment]
@ClarkxlYT This is not a pull request if it was it would have something saying add pull request.

**Comment by Luke18033** [comment]
@Konclan Yes, I actually think it is very annoying how people will put [New Item] or [Bug] at the beginning of issue titles when they could just add a label.

**Comment by LautaroL20** [comment]
@Luke18033 - Its funny, because you done the same in the past:

https://github.com/TeamSpen210/BEE2-items/issues/594
https://github.com/TeamSpen210/BEE2-items/issues/597

**Comment by ZombiePwner1** [comment]
LOL

**Comment by TheDarkBomber** [comment]
@Luke18033 Not most people can actually add labels.

**Comment by Luke18033** [comment]
@LautaroL20 Yes, but if you look at some of my other issues I have removed that, thx for showing me the ones I missed.

**Comment by Luke18033** [comment]
This would probably be better as a Start Reversed on cameras.

**Comment by TeamSpen210** [comment]
It'll be a separate item - among other things, in Old Aperture it should become a loudspeaker instead. It should also probably be knockable off walls.

**Comment by ZombiePwner1** [comment]
I recommend it should be one of those rectangular cameras they have in parking lots for old aperture.

**Comment by Konclan** [comment]
The old cctv security camera looking one. That wouldn't work because it's locked at an orientation and would look very strange if you tried to make it rotate

**Comment by ZombiePwner1** [comment]
Actually some of them can move.

**Comment by Luke18033** [comment]
This seems like it would be easy to implement, at least for Clean and Overgrown. Why hasn't it been added yet?

**Comment by Luke18033** [comment]
For Old Aperture, the camera should automatically pan back and forth instead of facing the player. It would still be knockable off walls.
